examples/jaeger-tracing/front-envoy-jaeger.json

```diff
@@ -0,0 +1,85 @@
+{
+  "listeners": [
+    {
+      "address": "tcp://0.0.0.0:80",
+      "filters": [
+        {
+          "type": "read",
+          "name": "http_connection_manager",
+          "config": {
+            "generate_request_id": true,
+            "tracing": {
+              "operation_name": "ingress"
+            }, 
+            "codec_type": "auto",
+            "stat_prefix": "ingress_http",
+            "route_config": {
+              "virtual_hosts": [
+                {
+                  "name": "backend",
+                  "domains": ["*"],
+                  "routes": [
+                    {
+                      "timeout_ms": 0,
```

Comment:
i'd probably remove timeout_ms setting and also for simplicity you can say "prefix": "/".